tests/integration/test_data_row_media_attributes.py

```diff
@@ -5,6 +5,8 @@ def test_export_empty_media_attributes(configured_project_with_label):
     project, _, _, _ = configured_project_with_label
     # Wait for exporter to retrieve latest labels
     sleep(10)
-    labels = project.label_generator()
-    label = next(labels)
-    assert label.data.media_attributes == {}
+    labels = project.label_generator().as_list()
+    assert len(
+        labels
+    ) == 1, "Labels could not be fetched via `project.label_generator()`"
```

Comment:
I am already imagining this error message being posted all over slack. This makes it seem like the problem is the sdk. I would say that label export job unexpectedly returned an empty result set.

Reply:
Thanks for the suggestions! Made the changes